pages/ksphere/kommander/latest/operations/cloud-providers/index.md

```diff
@@ -144,7 +125,49 @@ Below is the minimal IAM policy required:
 }
 ```
 
-#### Fill out the rest of the form
+### Role-based Authentication Method
+
+<p class="message--note"><strong>NOTE: </strong>The Role authentication method can only be used if your management cluster is running in AWS.</p>
+
+To utilize this option you going to need to attach the role to the instances your Kommander cluster is running. Unless you created the Kommander cluster [using the Instance Profile](/ksphere/konvoy/latest/install/install-aws/advanced-provisioning/#iam-instance-profiles) already.
+
+#### Fill out the form
+
+1. Fill out a display name for your cloud provider for later reference
+2. Fill out the Role ARN
+3. Optionally fill out the [External ID][external_id]
```

Comment:
Please add: `You want to do that if you share the role with a 3rd party and to secure your environment from accidentally used roles.`

Reply:
See https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html. Maybe also add this as a link.

Reply:
👍